event_seat_reserve/models/event_registration.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+
+from odoo import fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = "event.registration"
+
+    state = fields.Selection(
+        selection_add=[("reserved", "Reserved")], ondelete={"reserved": "set default"}
```

Comment:
```suggestion
        selection_add=[("reserved", "Reserved"), ("open",)], ondelete={"reserved": "set default"}
```

to choose the position, so that its in between `draft` and `open`

Reply:
Ohh thanks ! I didn't know it could be done like this.